apps/server/src/modules/oauth/uc/oauth.uc.ts

```diff
@@ -126,4 +127,31 @@ export class OauthUc {
 		const oAuthError: OAuthProcessDto = this.oauthService.getOAuthErrorResponse(error, provider);
 		return oAuthError;
 	}
+
+	async migrateUser(currentUserId: string, query: AuthorizationParams, systemId: string): Promise<UserMigrationDto> {
```

Comment:
I find it very hard to read this function... to the point that im having trouble giving you concrete hints what to change.

It might help to adjust the variable names to make clear what are old values, and what are new/changed/migrated values.

You can also think if there are any functions that can be extracted to be able to give them names.

Reply:
changed it.